cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,45 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+
+      cy.setLanguage();
+      cy.visit('/');
+    });
+  });
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@userId');
+});
+Given('they have the role of Application administrator', () => {});

```

Comment:
skal det være en sjekk på rolle her også (fra backend), eller skal det bare baseres på menyelementer?
Rollen kommer til å ligge i redux, kan være greit med en TODO-kommentar i så fall så det ikke blir glemt

Reply:
Min tanke var at det skal baseres på menyelementer for å frikoble end-til-ende-testingen fra implementasjonen. Har vel ikke tilgang til redux uten å trikse til ting?

Reply:
nei, det er sant!